webclient/app/src/app/features/apps/entities/entityDiagram/EntityDiagram.jsx

```diff
@@ -43,39 +49,37 @@ function EntityDiagram(props) {
         if (!editable) {
             return;
         }
-
         return entityRest.delete(entityId)
             .then(reloadEntities);
     }
 
-    function prepareUpdateEntity(updatedEntityInput, shallReloadEntities = true) {
-        const updatedEntity = {...updatedEntityInput};
+    function updateEntity(entity) {
+        return entityRest.createEntityByApp(appId, entity);
+    }
 
+    function reloadEntities() {
+        return entityRest.findAllEntitiesByApp(appId).then(response => {
+            const newEntities = response.data;
+            onChange(newEntities);
+            return newEntities;
+        });
+    }
+
+    function prepareUpdateEntity(updatedEntity) {
         if (!editable) {
             return;
         }
-
-        if (updatedEntity.isNewEntity) {
-            delete updatedEntity.id;
-            updatedEntity.isNewEntity = false;
-        }
-
-        if (currentEntity) {
-            setCurrentEntity(updatedEntity);
-        }
```

Comment:
Wir ziehen das ja in der find Funktion unten in dem Dialog raus, ne? Also irgendwas hab ich entweder falsch gedacht damals oder irgendwas nicht richtig umgestellt... Das sieht so aus als hätte ich die Entität da live in dem CurrentEntity State bearbeitet... Naja, so geht auch ^^